packages/core/src/App/Containers/RealAccountSignup/modal-login-prompt.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Button } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { redirectToLogin, redirectToSignUp } from '_common/base/login';
+import 'Sass/modal-login-prompt.scss';
+
+const ModalLoginPrompt = () => (
+    <div className='modal-login-prompt'>
+        <h2>{localize('This is only available for existing clients.')}</h2>
+        <p>{localize('If you have an active account, please log in to continue. Otherwise, please sign up.')}</p>
+
+        <div>
+            <Button secondary={true} text={localize('Log In')} onClick={redirectToLogin} />
+            <Button primary={true} text={localize('Sign Up')} onClick={redirectToSignUp} />
```

Comment:
For props that are simple booleans you only have to pass the prop name.

```suggestion
            <Button secondary text={localize('Log In')} onClick={redirectToLogin} />
            <Button primary text={localize('Sign Up')} onClick={redirectToSignUp} />
```